python/packages/autogen-ext/src/autogen_ext/models/openai/_transformation/registry.py

```diff
@@ -87,13 +87,14 @@ def _find_model_family(api: str, model: str) -> str:
     Finds the best matching model family for the given model.
     Search via prefix matching (e.g. "gpt-4o" → "gpt-4o-1.0").
     """
-    for family in MESSAGE_TRANSFORMERS[api].keys():
-        if model.startswith(family):
-            return family
-    return "default"
+    family = "unknown"
+    for _family in MESSAGE_TRANSFORMERS[api].keys():
+        if model.startswith(_family):
+            family = _family
+    return family
 
 
-def get_transformer(api: str, model_family: str) -> TransformerMap:
+def get_transformer(api: str, model: str, model_family: str) -> TransformerMap:
```

Comment:
I don't see `model` gets used here in this function.

Reply:
I fixed it. And add test case for what I want to that behavior,